src/components/Chart.js

```diff
@@ -0,0 +1,175 @@
+import React from "react";
+import axios from "axios";
+import Highcharts from "highcharts/highstock";
+import HighchartsReact from "highcharts-react-official";
+import { useState, useEffect } from "react";
+
+export default function Chart() {
+  const [chartData, setChartData] = useState([]);
+  const [time, setTime] = useState("10m");
+
+  useEffect(() => {
+    const fetchData = async function () {
+      const res = await axios.get(
+        `https://api.bithumb.com/public/candlestick/BTC_KRW/${time}`
+      );
+
+      await axios.post("http://localhost:8000/coins/candlestick", {
+        headers: {
+          candlestick: res.data.data,
+        },
+      });
+
+      setChartData(res.data.data);
+    };
+
+    fetchData();
+  }, [time]);
+
+  const ohlc = [];
+  const volume = [];
+  const groupingUnits = [
+    ["minute", [5, 10, 30, 60]],
+    ["hour", [1, 2, 4, 6, 12, 24]],
+    ["day", [1]],
+    ["week", [1]],
+    ["month", [1, 2, 3, 6]],
+  ];
+
+  for (let i = 0; i < chartData.length; i++) {
+    ohlc.push([
+      chartData[i][0],
+      Number(chartData[i][1]),
+      Number(chartData[i][2]),
+      Number(chartData[i][3]),
+      Number(chartData[i][4]),
+    ]);
+    volume.push([chartData[i][0], Number(chartData[i][5])]);
+  }
```

Comment:
```suggestion
    ]);
    
    volume.push([chartData[i][0], Number(chartData[i][5])]);
  }
```
한 칸 띄워주면 가독성이 조금 더 나아질 거 같습니다 !